packages/staking/src/features/BrowsePools/StakePoolsTable/StakePoolsGrid/StakePoolsGrid.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-magic-numbers */
+import { Box, Text } from '@lace/ui';
+import { SortField } from 'features/BrowsePools/types';
+import debounce from 'lodash/debounce';
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useMediaQuery } from 'react-responsive';
+import { ListRange } from 'react-virtuoso';
+import useResizeObserver from 'use-resize-observer';
+import { StakePoolCardSkeleton } from '../../StakePoolCard';
+import { StakePoolsListRowProps } from '../StakePoolsList/types';
+import { Grid } from './Grid';
+import * as styles from './StakePoolsGrid.css';
+import { StakePoolsGridItem } from './StakePoolsGridItem';
+
+const DEFAULT_DEBOUNCE = 200;
+
+const gridCardHeight = 84;
```

Comment:
```suggestion
const GRID_CARD_HEIGHT = 84;
```
for consistency as a top level const

Reply:
Done